examples/requirements.txt

```diff
@@ -1 +1,2 @@
 opencv-python
+depthai
```

Comment:
This is not enough to upgrade to latest depthai version.
Let's remove requirements.txt, it might not be trivial for everyone how to run, and it's easy to miss out `-r` from `python3 -m pip install -r requirements.txt` which causes confusion, and replace with `install_requirements.py` which install 3rd party dependencies (opencv-python), together with depthai lib, see comments below for suggestion.